--- origsrc/mp4v2-1.9.1/configure.ac	2009-07-13 18:22:46.000000000 -0500
+++ src/mp4v2-1.9.1/configure.ac	2009-07-27 19:14:28.133885000 -0500
@@ -95,8 +95,6 @@ AC_MSG_CHECKING([$PACKAGE_NAME platform 
 X_PLATFORM=posix
 case ${host} in
 *-*-cygwin)
-    # do not support shared
-    enable_shared=no
     X_CXX_W="$X_CXX_W -Wno-format"
     if test "$enable_cygwin_win32" = "yes"; then
         X_PLATFORM=win32
@@ -206,6 +204,9 @@ case ${host} in
     *-*-darwin*)
         X_libmp4v2_la_LDFLAGS='-Wl,-current_version,$(PROJECT_version_major).$(PROJECT_version_minor).$(PROJECT_version_point) -Wl,-compatibility_version,$(PROJECT_version_major).0.0'
         ;;
+    *-*-cygwin*)
+        X_libmp4v2_la_LDFLAGS='-no-undefined'
+        ;;
     *)
         X_libmp4v2_la_LDFLAGS=
         ;;
--- origsrc/mp4v2-1.9.1/doc/GNUmakefile.mk	2009-07-13 18:27:48.000000000 -0500
+++ src/mp4v2-1.9.1/doc/GNUmakefile.mk	2009-07-27 19:27:13.264076000 -0500
@@ -200,7 +200,7 @@ siteclean:
 ###############################################################################
 
 $(DOC.man.out): | $(dir $(DOC.man.out))
-$(DOC.man.out): $(DOC.out/)man/man1/%.1: $(BUILD/)%
+$(DOC.man.out): $(DOC.out/)man/man1/%.1: $(BUILD/)%$(EXEEXT)
 	$(call HELP2MAN.cmd,$<,$@)
 
 $(DOC.texi2html.out): $(DOC.texi.includes) | $(dir $(DOC.texi2html.out))
